### ISSUE TITLE
missing or unclear focus indicator for non-touch devices

### ISSUE THREAD

**Issue by aconitumh** [signalapp/Signal-Android#14254]
### Guidelines

- [x] I have searched searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

### Bug description

As described in title i have stumbled upon the following issues separated by screens, there are probably more affected.
Issues occur in both dark an light mode 

Start Screen:
Own profile picture: Not focusable at all (no indicator, still functioning).
Other profile pictures: Show visible focus indicator when navigated with D-pad.
Chats: Only become focusable after navigating all the way down and then pressing “up” from picture/camera icon.
Even then, only the currently visible chats can be focused.
Chats that were previously scrolled out of view (e.g. topmost ones) are unable to get back into view unless closing and reopening and then it's just same thing over again
(Note: Navigation issue, might be a different ticket.)

Profile Picture Screen:
Message and Call only focused when navigated through D-right/D-left from profile picture, Video only when navigated from PP-Name-Video.
Switching between them does not show any focus, they are all functioning regardless. 
Just gotta pay a lot of attention to what you have clicked.

Chat Screen
Send & "+" button: functional but barely visible, it's really just guessing if I'm hitting send or opening the more menu at the top of chat or not doing anything at all.
Emoji/Sticker tabs:
Focus indicators exist, but are extremely faint
Switching tabs shows almost no visual feedback and partially impossible (navigation issue)
Long-press message menu:
Menu appears visually but focus indicator for menu options is completely missing.
(Navigation actually continues in background,ol also different issue albeit related)

Actual results:
No focus indicator at all, or focus indicators that are too subtle to detect leading to poor accessibility and guesswork navigation.

Expected results:
All interactive elements should have clearly visible and consistent focus indicators when navigated via D-pad or accessibility tools.  Focus should never be lost or invisible, and users should always be able to determine which item is currently selected without relying on guesswork or external input devices.

I have been able to get around most issues with the built in pointer of the phone, tho after updating signal or rebooting the phone does not offer the option for the pointer in signal anymore making proper focus indicators (as well as navigation) crucial for accessibility and usability.

### Device

KYOCERA A203KC

### Android version

10

### Signal version

7.48.1
